app/tests/test_memory.py

```
@@ -0,0 +1,369 @@
+"""Tests for memory context management system."""
+
+import json
```

Ruff (0.11.9)

`json` imported but unused

Remove unused import: `json`

(F401)